package/yast2-iscsi-client.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Nov 06 10:00:14 CET 2018 - aschnell@suse.com
+
+- enable rubocop
+- 4.1.2
```

Comment:
I think you don't have to update the version (and release a new package) if you plan to work on it later. (According to https://trello.com/c/f7Gnda6X/443-sles12-sp3-1045139-sles-12-sp3-change-behaviour-of-yast-when-using-iscsi-luns there is planned some behavior change.)

And IIRC Jenkins would complain about missing FATE/bugzilla number. If you later add some real fix (with a bug number) then you do not have to create a dummy bug report just for this Rubocop change.